extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -35,6 +35,14 @@ define('ext.wikia.adEngine.video.uapVideo', [
 		};
 	}
 
+	function shouldUseMegaAdUnitBuilder(type) {
+		var uapTypesMap = {
```

Comment:
You can define this map outside of function body. This way you avoid creating it on every function call.